### PR TITLE
Add attribute to Vendors DB entity

### PR DESCRIPTION
**Pull request by taerim25**
# Description

-City attribute was added to Vendors DB entity.
-Insertion method of city entry was updated to reflect entity change
-Test was updated to reflect entity change
-Request body in main,py was updated to reflect entity change

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.  
Provide instructions so we can **reproduce**.  
Please also list any relevant details for your test configuration

- [x] python3.6 -m unittest discover tests/api
Ran unittest to see if new format of entry was correctly inserted
- [x] manual addition of entry
Test entry was correctly added to the database when checking in psql commandline.
# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I will rebase before merging

### CODE REVIEW

**Comment by taerim25** [comment]
[Add attribute to Vendors DB entity](https://trello.com/c/ByqzUjRj/38-add-attribute-to-vendors-db-entity)

File: backend/app/main/dbase.py [hunk 1]
```diff
@@ -66,7 +67,7 @@ class Food(db.Entity):
 # INSERTIONS
 @db_session
 def create_vendor(vendor):
-    v = Vendor(name=vendor.name, address=vendor.address)
+    v = Vendor(name=vendor.name, address=vendor.address,city =vendor.city)
```
Comment (yuedai0531) [inline backend/app/main/dbase.py]: remove space between `city =`

Comment (yuedai0531) [reply backend/app/main/dbase.py]: also why do we have a v in front?

**Comment by TamPhilip** [comment]
Resolve conflits

**Pull request merged**